src/coreclr/jit/instr.h

```diff
@@ -474,6 +474,7 @@ enum emitAttr : unsigned
                 EA_BYREF         = EA_BYREF_FLG |  EA_PTRSIZE,       /* size == -2 */
                 EA_DSP_RELOC_FLG = 0x400, // Is the displacement of the instruction relocatable?
                 EA_CNS_RELOC_FLG = 0x800, // Is the immediate of the instruction relocatable?
+                EA_CNS_SEC_RELOC = 0x1000,
```

Comment:
Should this be:
```suggestion
                EA_SEC_RELOC_FLG = 0x1000,
```
to more closely match the others?

Can you add a comment?